### PR TITLE
Improve consistency in MathJax examples.

### PR DESCRIPTION
**Pull request by drupol**
### Component/Part

The [MathJax example in Features page](https://demo.hedgedoc.org/features#MathJax):

image

### Description

This PR improve consistency in the MathJax example:

image

### Steps

- [x] Added / updated documentation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `master` for 1.x & docs, `develop` for 2.x

### CODE REVIEW

File: public/docs/features.md [hunk 1]
```diff
@@ -325,7 +325,7 @@ gantt
   section Section
   A task: a1, 2014-01-01, 30d
   Another task: after a1, 20d
-
```
Comment (DerMolly) [inline public/docs/features.md:328]: Did you mean to change this?

**Review by DerMolly — COMMENTED** [review]
LGTM

File: public/docs/features.md (hunk 1 above)
Comment (drupol) [reply public/docs/features.md:328]: No, this was done automatically by VSCode while editing the page.

Comment (DerMolly) [reply public/docs/features.md:328]: Then I think you should remove this…

Comment (drupol) [reply public/docs/features.md]: Fixed.

**Review by ErikMichelson — APPROVED** [review]

**Review by DerMolly — APPROVED** [review]

**Comment by DerMolly** [comment]
Thank you, @drupol

**Pull request merged**